cpp/src/io/parquet/rle_stream.cuh

```diff
@@ -154,154 +176,94 @@ struct rle_stream {
   static constexpr int run_buffer_size = rle_stream_required_run_buffer_size<decode_threads>();
 
   int level_bits;
```

Comment:
thoughts to having prefixes/suffixes to help tell data members apart from local variables?
not too relevant of this PR

Reply:
In the past I've use m_ for member variables. Not sure I really loved it though.